boards/st/sensortile_box_pro/Kconfig.defconfig

```diff
@@ -20,46 +20,7 @@ config SPI_STM32_INTERRUPT
 	default y
 	depends on SPI
 
-if BOARD_SERIAL_BACKEND_CDC_ACM
-
-config USB_DEVICE_STACK
-	default y
-
-config USB_CDC_ACM
-	default SERIAL
-
-config USB_DEVICE_INITIALIZE_AT_BOOT
-	default y if CONSOLE
-
-config SHELL_BACKEND_SERIAL_CHECK_DTR
-	default SHELL
-	depends on UART_LINE_CTRL
-
-config UART_LINE_CTRL
-	default SHELL
-
-config USB_DEVICE_REMOTE_WAKEUP
-	default n
-
-config USB_DEVICE_VID
-	default 0x0483
-
-config USB_DEVICE_PID
-	default 0x5740
-
-config USB_DEVICE_PRODUCT
-	default "Zephyr CDC SensorTile.box PRO"
```

Comment:
These 3 symbols look like legit board specific configuration.
If there a specific, documented, reason to remove them?

Reply:
rather other way around. Added to commit message:
```
    Do not adapt board-specific configurations such as unknown PID/VID or
    string descriptors. There is no justification for using them on specific
    boards, and we do not have formal approval to use them in the project
    tree. Also, we need more uniform configuration, since the main reason
    for enabling CDC ACM here is to allow users to run examples like
    hello_world right out of the box. Of course, anyone is free to customize
    these settings in their fork or downstream project.
```